runtime/drive_handler.go

```diff
@@ -107,6 +107,10 @@ func (h *StubDriveHandler) Reserve(
 	if len(h.freeDrives) == 0 {
 		return nil, ErrDrivesExhausted
 	}
+	if _, ok := h.usedDrives[id]; ok {
+		// This case means that driver wasn't released or removed properly
+		return nil, fmt.Errorf("drive with ID %s already in use, try to release it")
```

Comment:
I think this is missing the `id` to fill in `%s` with. Also, while `try to release it` is good in that it's trying to guide the user to fixing the problem, I'm not sure it's going to mean much to someone unfamiliar with the internals of our code. Maybe just including something similar to the comment above this line like `drive with ID %s already in use, a previous attempt to remove it may have failed` would be better in that it could guide users to their logs to see a previous failure or something like that.